### ISSUE TITLE
[enhancement] Add GUI

### ISSUE THREAD

**Issue by piymis** [rohitk-singh/WhatsApp-Square-DP#7]

**Comment by rohitk-singh** [comment]
Any opinions on which framework to go with? PyQt or Kivy or TclTk? (or less preferred WxWidgets/PyGtk?)

**Comment by piymis** [comment]
I think for our purpose Tkinter would be good enough. Shipped with python, thus no major dependency and avoiding the complexity of other frameworks at the same time. What do you say?

**Comment by rohitk-singh** [comment]
I would prefer Tkinter or Kivy. Tkinter due to no dependency. And Kivy due to the fact that we can build & run it on Android.

**Comment by piymis** [comment]
IMO portability to other platforms should be a separate thing. Tkinter should be a go then.

**Comment by rohitk-singh** [comment]
Can we just use this and be done with https://github.com/chriskiehl/Gooey

**Comment by piymis** [comment]
Looks good to me. Want to have a discussion on the implementation road-map?

**Comment by piymis** [comment]
BTW, before progressing with GUI development, it's better to have a design
ready for the final UI. What do you think?

**Comment by rohitk-singh** [comment]
Yeah we can discuss implementation. Do you have any specific layout in
mind? Or, aware of any existing apps/program for inspiration?
